bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/PJLinkDeviceHandler.java

```diff
@@ -101,17 +106,24 @@ public void refresh(PJLinkDeviceConfiguration config) {
     }
 
     PJLinkDeviceHandler.this.logger.debug("Polling device status...");
-    if (config.refreshPower) {
+    if (config.refreshPower && PJLinkDeviceHandler.this.getThing().getStatusInfo().getStatus() == ThingStatus.ONLINE) {
```

Comment:
If I am not wrong, you can get the status directly with `getThing().getStatus()`